### ISSUE TITLE
[Bug]: Doesn't show notes

### ISSUE THREAD

**Issue by MitchWagnerSD** [Quorafind/Obsidian-Daily-Notes-Editor#37]
### Bug Description

I just installed this plugin and ran it. The first time I ran it, the most recent note it showed was for December 18, nearly two months ago. 

I shut down Obsidian, restarted it, and ran the command again. This time, the only note it showed was for Jan. 18, 2022, more than two years out of date. 

I shut down Obsidian, restarted, and ran Daily Note Editor again, and this time, the most recent note it showed was yesterday, so that's better I guess?

I see the last update for the plugin was a very long time ago, and I wonder whether it's still being supported. 

### Obsidian Version

1.5.5

### Which plugin are you using for daily notes?

- [X] Default Daily Note Plugin
- [ ] Periodic Notes Plugin

### Checklist

- [X] I updated to the latest version of the plugin.

**Comment by calvinwyoung** [comment]
@MitchWagnerSD I'm running into the same issue. Did you figure out a workaround?

**Comment by MitchWagnerSD** [comment]
@calvinwyoung No, I never have. I use the Daily Notes Viewer plugin instead. 

https://github.com/Johnson0907/obsidian-daily-notes-viewer

**Comment by hmijail** [comment]
I suspect this issue was a combination of 2 bugs:
1. Notes out of order
2. Only a few notes are shown

The first bug seems to have been fixed recently in release [0.1.5](https://github.com/Quorafind/Obsidian-Daily-Notes-Editor/releases/tag/0.1.5).
The second bug is still there: if I open the Daily Notes Editor view, only the latest 1-2 days are shown.

But I think I found the reason and a workaround: if the shown note/s are too short to cause the scrollbar to appear, then no more notes are included in the view.

The workaround then is to change the size of the window momentarily, making it narrow, so that the text gets flowed down the available space. Once the scrollbar appears, the rest of notes will be populated! At that point you can resize the window back to your preferred size and the viewer keeps working as intended.

**Comment by calvinwyoung** [comment]
@hmijail Can you try the version linked in this comment? It seems to fix the issue for me:
https://github.com/Quorafind/Obsidian-Daily-Notes-Editor/pull/41#discussion_r1653018811
